src/module.ts

```diff
@@ -158,66 +180,73 @@ export default defineNuxtModule<ModuleOptions>({
     // https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/preload
     // optionally increase loading priority
     if (options.preload) {
-      head.link.push({
-        key: 'gf-preload',
-        rel: 'preload',
-        as: 'style',
-        href: url
-      })
+      for (const url of urls) {
+
+        head.link.push({
+          key: 'gf-preload',
+          rel: 'preload',
+          as: 'style',
+          href: url
+        })
+      }
     }
 
     // append CSS
     if (options.useStylesheet) {
-      head.link.push({
-        key: 'gf-style',
-        rel: 'stylesheet',
-        href: url
-      })
-
+      for (const url of urls) {
+        head.link.push({
+          key: 'gf-style',
+          rel: 'stylesheet',
+          href: url
+        })
+      }
       return
     }
 
     if (isNuxt2()) {
       // JS to inject CSS
-      head.script.push({
+      for (const url of urls) {
+        head.script.push({
+          key: 'gf-script',
+          innerHTML: `(function(){var l=document.createElement('link');l.rel="stylesheet";l.href="${url}";document.querySelector("head").appendChild(l);})();`
+        })
+
+        // no-JS fallback
+        head.noscript.push({
+          key: 'gf-noscript',
+          innerHTML: `<link rel="stylesheet" href="${url}">`,
+          tagPosition: 'bodyOpen'
+        })
+
+        // Disable sanitazions
+        // @ts-ignore
+        head.__dangerouslyDisableSanitizersByTagID = head.__dangerouslyDisableSanitizersByTagID || {}
+        // @ts-ignore
+        head.__dangerouslyDisableSanitizersByTagID['gf-script'] = ['innerHTML']
+        // @ts-ignore
+        head.__dangerouslyDisableSanitizersByTagID['gf-noscript'] = ['innerHTML']
+      }
+      return
+    }
+
+    // JS to inject CSS
+    for (const url of urls) {
+      head.script.unshift({
         key: 'gf-script',
-        innerHTML: `(function(){var l=document.createElement('link');l.rel="stylesheet";l.href="${url}";document.querySelector("head").appendChild(l);})();`
+        children: `(function(){
+          var h=document.querySelector("head");
+          var m=h.querySelector('meta[name="head:count"]');
+          if(m){m.setAttribute('content',Number(m.getAttribute('content'))+1);}
+          else{m=document.createElement('meta');m.setAttribute('name','head:count');m.setAttribute('content','1');h.append(m);}
+          var l=document.createElement('link');l.rel='stylesheet';l.href='${url}';h.appendChild(l);
+        })();`
```

Comment:
Maybe create a script to create the stylesheet.
In case of several url this script will be duplicated.

Reply:
so is it possible rn?